lms/static/scripts/frontend_apps/components/dashboard/SyncGradesButton.tsx

```diff
@@ -116,11 +116,7 @@ export default function SyncGradesButton({
       );
     }
 
-    if (studentsToSync.length > 0) {
-      return `Sync ${studentsToSync.length} grades`;
-    }
-
-    return 'Grades synced';
+    return `Sync ${studentsToSync.length} grades`;
```

Comment:
I removed one case here, since this content won't be rendered at all when `allGradesAreSynced` is true.

That avoids an untestable code branch.

Reply:
Separate from the change in this PR, I was reminded in a different PR that we need to adjust pluralization depending on the grade count.

Reply:
I took the opportunity to address this. I also thought about it a couple times and was always focused on something else.